OpenEdXMobile/res/layout/row_discussion_comments_comment.xml

```diff
@@ -1,17 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.v7.widget.CardView
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
#JustAutoFormatThings. Keeping it as is, cuz whenever/if the auto-format will be done in future, this will surface again.